OutOfSchool/OutOfSchool.RazorTemplatesData/Services/RazorViewToStringRenderer.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Abstractions;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewEngines;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Routing;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OutOfSchool.RazorTemplatesData.Services
+{
+    public class RazorViewToStringRenderer : IRazorViewToStringRenderer
+    {
+        private readonly IRazorViewEngine _viewEngine;
```

Comment:
Private field naming seems to be inconsistent to other code in project. Please confirm if underscore (_) should be used as prefix.